### PR TITLE
Addnote page

### LINKED ISSUE #100: Create "Note" screen (for adding note, editing note, and viewing note)
Currently when user double-clicks the note, a new node is added to the mind map, and an editor is shown to edit the new note name:

image

But according to [the new design](https://www.figma.com/file/FxmwqO6LRg7tOVXrZqMMG0/Neural-Notes-Design-v.0.3?node-id=212%3A836) the "add note" should be a separate screen (and the same screen serves as "edit note" and "view note"):

image

So when user double-clicks a note to create a new note, we should change page to "note page", instead of showing an editor on the same page.

What is to be done:
1. Create a new page "Note page". This page will be used for adding a new note, editing a note, and for viewing a note.
2. When user double-clicks a note, redirect to the "Note page".
3. Move `NoteDetailsContainer` from the MindMap page to the "Note page"

image

### PR DESCRIPTION
**Pull request by kvoncode**
Rebased Working NoteDetailsPage, without yarn.lock Closes #100 

- arrow button onclick goes to notes tree page
- fixed styles

Please review and comment.

### CODE REVIEW

File: src/components/LoginPage/LoginPageComponent.js [hunk 1]
```diff
@@ -89,7 +89,9 @@ const Button = css`
 
 const GoogleLoginButton = styled.button`
   ${Button}
-  float: right;
+  position: absolute;
+  right: 2rem;
+  top: 1rem;
```
Comment (DarthVanger) [inline src/components/LoginPage/LoginPageComponent.js:94]: What is this for? Did we have any problems with the login button on the login page?

@TomaTomoe pls review.

File: src/components/NoteDetailsPage/NoteDetailsPage.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  };
+  return (
+    <Wrapper>
+      <BackButtonWrapper>
+        <IconButton onClick={handleClick} aria-label="back">
+          <ArrowBackIcon style={{ fill: '#E4E3E4' }} />
+        </IconButton>
+      </BackButtonWrapper>
+      <NoteDetailsContainer />
+      {/* <StyledButton onClick={handleClick}>View/Edit Note</StyledButton>; */}
```
Comment (DarthVanger) [inline src/components/NoteDetailsPage/NoteDetailsPage.js:33]: What is this for? Which `StyledButton` is this?

**Comment by DarthVanger** [comment]
Please delete `yarn.lock`, we don't need it.

**Comment by DarthVanger** [comment]
Please resolve the merge conflicts.

File: src/components/LoginPage/LoginPageComponent.js (hunk 1 above)
Comment (kvoncode) [reply src/components/LoginPage/LoginPageComponent.js:94]: It adds a little margin on top

File: src/components/NoteDetailsPage/NoteDetailsPage.js (hunk 2 above)
Comment (kvoncode) [reply src/components/NoteDetailsPage/NoteDetailsPage.js:33]: It's not needed and commented. Came from copy paste of notes tree page

**Comment by kvoncode** [comment]
> Please delete `yarn.lock`, we don't need it.

Please add yarn.lock, it speeds up development a lot

**Comment by kvoncode** [comment]
> Please resolve the merge conflicts.

Ok

**Pull request closed without merging**